### PR TITLE
Added weapons short keys support.

### PR DESCRIPTION
**Pull request by RICCIARDI-Adrien**

### CODE REVIEW

**Comment by TeslaRus** [comment]
Thanks for PR, I will see it little bit later...
P.S. first look - all is ok.

File: src/weapons.cpp [hunk 1]
```diff
@@ -1,5 +1,6 @@
 
 #include "audio/audio.h"
+#include "controls.h"
```
Comment (TeslaRus) [inline src/weapons.cpp]: hmmm ok, I know how to do it without this include )

Comment (RICCIARDI-Adrien) [reply src/weapons.cpp]: Hi !
How could this have been done ?
I'm interested in OpenTomb internal architecture.

Comment (TeslaRus) [reply src/weapons.cpp]: done in my commit - just substract weapon code outside function, that uses it  ))

**Pull request merged**